src/tests/JIT/HardwareIntrinsics/X86/Sse2/Program.Sse2.cs

```diff
@@ -5,6 +5,7 @@
 using System.Collections.Generic;
 
 [assembly:Xunit.ActiveIssue("https://github.com/dotnet/runtime/issues/75767", typeof(TestLibrary.PlatformDetection), nameof(TestLibrary.PlatformDetection.IsMonoLLVMAOT))]
+[assembly:Xunit.ActiveIssue("https://github.com/dotnet/runtime/issues/75767", typeof(TestLibrary.PlatformDetection), nameof(TestLibrary.PlatformDetection.IsMonoFULLAOT))]
```

Comment:
```suggestion
[assembly:Xunit.ActiveIssue("https://github.com/dotnet/runtime/issues/102150", typeof(TestLibrary.PlatformDetection), nameof(TestLibrary.PlatformDetection.IsMonoFULLAOT))]
```
Not sure about the preference, but it might be better to mention the new issue. Additionally, https://github.com/dotnet/runtime/issues/75767 has been already closed an replaced by https://github.com/dotnet/runtime/issues/91392.